binder/environment.yml

```diff
@@ -7,5 +7,6 @@ dependencies:
 - isort
 - jupyterlab=0.35
 - yapf
+- rpy2
```

Comment:
Maybe we should add `r-formatr` and whatever packages needed for a functional R environment in binder? (Not that fussed, you can opt not to do this if you don't want to)

Reply:
Is the binder will enforce them to be installed in the conda environment? r-formatr is available only from conda R repo.

Reply:
> Is the binder will enforce them to be installed in the conda environment?
Hmm [binder](https://mybinder.org) is a tool we use to let user test out the plugin without installing it themselves, it doesn't the end user if they are installing it on their machine.
> r-formatr is available only from conda R repo
I think we can add the r channel in the channels section of `environment.yml` and it should be fine, we are already using `conda-forge` which is not the default channel.

Reply:
Make sense.